src/js/base/module/VideoDialog.js

```diff
@@ -86,7 +138,30 @@ export default class VideoDialog {
     const fbMatch = url.match(fbRegExp);
 
     let $video;
-    if (ytMatch && ytMatch[1].length === 11) {
+    let urlVars = '';
+    const $videoAspect = this.$dialog.find('.note-video-aspect');
+    const $videoQuality = this.$dialog.find('.note-video-quality');
+    const $videoSuggested = this.$dialog.find('.note-video-suggested input[type=checkbox]');
+    const $videoCaptions = this.$dialog.find('.note-video-captions input[type=checkbox]');
+    const $videoControls = this.$dialog.find('.note-video-controls input[type=checkbox]');
+    const $videoAutoplay = this.$dialog.find('.note-video-autoplay input[type=checkbox]');
+    const $videoLoop = this.$dialog.find('.note-video-loop input[type=checkbox]');
+
+    const selectVideoAspect = $videoAspect.val();
+    const selectVideoQuality = $videoQuality.val();
+    const checkVideoSuggested = $videoSuggested.is(':checked');
+    const checkVideoCaptions = $videoCaptions.is(':checked');
+    const checkVideoControls = $videoControls.is(':checked');
+    const checkVideoAutoplay = $videoAutoplay.is(':checked');
+    const checkVideoLoop = $videoLoop.is(':checked');
+    const vWidth = 788.54;
+    const vHeight = 443;
+    const vQuality = '';
+
+    if (selectVideoAspect == '4-3') vWidth = 589.19;
```

Comment:
vWidth is const variable,  so it can not assign value.

Reply:
Should that be a Let?

Reply:
yes, it is right.